projects/graph/src/graph.py

```diff
@@ -8,8 +8,8 @@ def __init__(self):
         pass  # TODO
         self.vertices = {}
     
-    def add_vertex(self, vertex_id):
-        self.vertices[vertex_id] = Vertex(vertex_id)
+    def add_vertex(self, vertex_id, vertices):
+        self.vertices[vertex_id] = vertices(vertex_id)
```

Comment:
This line right here is saying, in the dictionary vertices, the key value corresponding to the key 'vertex_id' is 'vertices(vertex_id)'. I'm not sure what this key value 'vertices(vertex_id)' is supposed to return.